.projectile

```diff
@@ -0,0 +1 @@
+-/vendor
```

Comment:
I think you want to add `.projectile` to your global `~/.gitignore`, and then you can sprinkle these projectile ignore files anywhere.